File: app/src/main/java/com/example/pelisapp/FilmModel.java
package com.example.pelisapp;

public class FilmModel {
    int image;
    String title;
    String info;

    public FilmModel(int image, String title, String info) {
        this.image = image;
        this.title = title;
        this.info = info;
    }

    public int getImage() {
        return image;
    }

    public String getTitle() {
        return title;
    }

    public String getInfo() {
        return info;
    }
}

File: app/src/main/java/com/example/pelisapp/models/FilmModel.kt
package com.example.pelisapp.models
import android.os.Parcelable
import kotlinx.parcelize.Parcelize
@Parcelize
data class FilmModel(
        val image: String,
        val title: String,
        val info: String,
        val id: Int=0
): Parcelable
